src/sql/azdata.proposed.d.ts

```diff
@@ -1870,10 +1870,8 @@ declare module 'azdata' {
 		GreaterThanOrEquals = 5,
 		Between = 6,
 		NotBetween = 7,
-		Contains = 8,
-		NotContains = 9,
-		IsNull = 10,
```

Comment:
Why are the null ones being removed? No use case currently or something?

Reply:
Yeah, they aren't used anywhere right now.